docs/tutorials/simulations/MMsbm.ipynb

```diff
@@ -0,0 +1,170 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Mixed Membership Stochastic Blockmodel(MMSBM)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A [Stochastic Block Model(SBM)](./sbm.ipynb) produces graphs containing communities: disjoint subgraphs characterized by differing edge probabilities for vertices within and between communities. \n",
+    "\n",
+    "Given a network with $k$ number of communities, a SBM is parametrized by the number of vertices in each community $n$; a block connectivity matrix\t$B\\in R^{kxk}$ where each $(i^{th}, j^{th})$ entry specifies the probability of connection between nodes pertaining to the $i^{th}$ community and the $j^{th}$ community; and a membership matrix $Z\\in R^{nxk}$ in which the $i^{th}$ row vector contains information regarding which of the $k$ communities the node $i$ pertains to. In fact, for any $i^{th}$ row of $Z$ each $j^{th}$ entry indicates the probability of the node $i$ pertaining to the community $j$. Each edge between nodes is then sampled from Bernoulli distribution according to a probability matrix $P\\in R^{nxn}$ such that $P$ = $Z B Z^{T}$ and each $(i^{th}, j^{th})$ entry indicates the probability of connection between node $i$ and $j$.\n",
+    "\n",
+    "One drawback of utilizing SBM is that each node must be assigned exclusevily to a single community. Therefore each row of $Z$ contains $1$ in the $j^{th}$ column indicating the community to which that node belongs to and $0$ for all the other entries. \n",
+    "\n",
+    "The Mixed Membership Stochaistic Block Model(MMSBM) addresses this issue by adding some variability in the community assignment process to each node. Indeed, given a network with $k$ communities and a connectivity matrix $B\\in R^{kxk}$ each node $p$ is assigned a membership vector $\\vec{\\pi}_p$. As aforementioned, the $i^{th}$ entry of this vector indicates the probability of node $p$ pertaining to the $i^{th}$ community when interactong with another node. Differently from SBM, a node is allowed to pertain to multiple communities. This is achieved by estimating how often a node act as belonging to a community with respect to another when forming connections with other nodes. For example consider:\n",
```

Comment:
a few typos here: when "interacting" with another node; how often a node "acts" as

Reply:
I am sorry that I have wasted some of your precious time on such silly mistakes. Will fix them right away. I read the tutorial and docs a couple of times and so I am not sure why I didn't catch those beforehand.

Reply:
no worries!